homeassistant/components/sia/alarm_control_panel.py

```diff
@@ -75,38 +106,41 @@ async def async_setup_entry(
 class SIAAlarmControlPanel(SIABaseEntity, AlarmControlPanelEntity):
     """Class for SIA Alarm Control Panels."""
 
+    entity_description: SIAAlarmControlPanelEntityDescription
+
     def __init__(
         self,
-        entry: ConfigEntry,
-        account_data: dict[str, Any],
-        zone: int,
+        port: int,
+        account: str,
+        zone: int | None,
+        ping_interval: int,
+        entry_id: str,
+        entity_description: SIAAlarmControlPanelEntityDescription,
     ) -> None:
         """Create SIAAlarmControlPanel object."""
-        super().__init__(entry, account_data, zone, DEVICE_CLASS_ALARM)
-        self._attr_state: StateType = None
-        self._old_state: StateType = None
+        super().__init__(port, account, zone, ping_interval, entity_description)
 
         self._attr_unique_id = SIA_UNIQUE_ID_FORMAT_ALARM.format(
-            self._entry.entry_id, self._account, self._zone
+            entry_id, account, zone
         )
+        self._attr_state: StateType = None
+        self._old_state: StateType = None
+        self._attr_supported_features = 0
```

Comment:
Since it is static, `self._attr_supported_features` should be defined at the class level:
```python
class SIAAlarmControlPanel(SIABaseEntity, AlarmControlPanelEntity):
    """Class for SIA Alarm Control Panels."""

    entity_description: SIAAlarmControlPanelEntityDescription
    _attr_supported_features = 0
```